### DISCUSSION TITLE
Where can I find the `<div class="deck">` element of the documentation on the Github repo

### DISCUSSION THREAD

**Discussion by adolfojmnz** [celery/celery#8184] [Q&A]
On every page of the celery documentation, this structure can be found. 

```html
<body>
  <div class="document">
    <div class="documentwrapper">
      <div class="bodywrapper">
        <div class="body" role="main">

          <div class="deck">
            <p>
              This document describes the current stable version of Celery (5.2).
              For development docs,
              <a href="https://docs.celeryq.dev/en/master/index.html">go here</a>.
            </p>
          </div>

          [...]

        </div>
      </div>
    </div>
  </div>
</body>
```

I can't find where that HTML document is stored or what generates it. 
Any help?

**Comment by ChihweiLHBird** [comment]
Yep, the link was broken. Wanted to fix it but couldn't find the place where it's generated or defined.
<img width="1122" alt="image" src="https://github.com/celery/celery/assets/43905414/bb9d8796-bf4a-481d-a07e-6ee223f248cc">

**Reply by adolfojmnz** [reply]
Yes, that is precisely what I wanted to do.
